src/Tokenizer/Tokens.php

```diff
@@ -1064,7 +1064,7 @@ public function isAllTokenKindsFound(array $tokenKinds): bool
     public function isAnyTokenKindsFound(array $tokenKinds): bool
     {
         foreach ($tokenKinds as $tokenKind) {
-            if (!empty($this->foundTokenKinds[$tokenKind])) {
+            if ($this->isTokenKindFound($tokenKind)) {
```

Comment:
this is very frequent call, we done some microoptimizations earlier for Tokens class to reduce hotpaths. maybe we do not need to call the method here

Reply:
If we _really_ don't want additional method call, then it should be changed to `isset($this->foundTokenKinds[$tokenKind])`, just like `isTokenKindFound()`'s implementation was changed in #7139 (which is weird, since it's a removal of `empty()` call and should be here 😝). But I believe this is negligible difference and we could call the method here.

Reply:
I've extracted removing `empty` in `isTokenKindFound` to separate PR as it was not that straightforward and I've expected some discussion.

What I did not expect is changes in `isAnyTokenKindsFound` and the other one -> I've reverted them here, so we can safely merge it and focus on `Tokens` in the other PR.

Reply:
We already did 😉  - `isset` as microoptimization